src/utils/polygon-migration.ts

```diff
@@ -7,5 +7,13 @@ export const metamaskUrl = () => "https://metamask.io/";
 
 export const migrationNetwork = () => import.meta.env.VITE_MIGRATION_NETWORK || "ark.devnet";
 
+export const polygonExplorerLink = () => import.meta.env.VITE_POLYGON_EXPLORER_URL || "https://mumbai.polygonscan.com";
+
+export const polygonTransactionLink = (transactionId: string) => `${polygonExplorerLink()}/tx/${transactionId}`;
+
+export const polygonContractAddress = () => import.meta.env.VITE_POLYGON_CONTRACT_ADDRESS;
```

Comment:
@ItsANameToo do you think it's necessary to add fallback contract address?

Reply:
is fine without default